tests/unit/types/argument-set-test.js

```diff
@@ -49,9 +56,19 @@ test("it can be made from a query", function(assert) {
   assert.equal(set[1].value, 10);
 });
 
-test("filters out null arguments", function(assert) {
+test("does not filter out null arguments from a query", function(assert) {
   let set = ArgumentSet.fromQuery({ status: null, limit: 10 });
```

Comment:
Yes, I know. It makes sure we will not filtered out in the future. It should be in this test though.